src/pkg/aws/ecs/tail.go

```diff
@@ -207,6 +201,61 @@ type tailEventStream interface {
 	Events() <-chan types.StartLiveTailResponseStream
 }
 
+type collectionStream struct {
+	streams []tailEventStream
+	ch      chan types.StartLiveTailResponseStream
+	errch   chan error
+	ctx     context.Context
+	lock    sync.Mutex
+}
+
+func (c *collectionStream) Add(s tailEventStream) {
```

Comment:
consider renaming this to reflect the fact that it starts a go-routine

Reply:
Renamed to `addAndStart`